app-nextjs-starter/pages/401.tsx

```diff
@@ -1,7 +1,8 @@
-import {NextPage} from "next";
-import {useEffect} from "react";
-
-import {Layout} from '@src/components/Layout'
+import { NextPage } from "next";
+import { useEffect } from "react";
+import { Layout } from "@src/components/Layout";
+import { UnauthorizedIcon } from "@src/components/UnauthorizedIcon";
+import styles from "../src/styles/401.module.css";
```

Comment:
```suggestion
import styles from "@src/styles/401.module.css";
```

Reply:
Fixed this import! 

Thanks, @eunjae-lee.